### ISSUE TITLE
Add more information about PMC members

### ISSUE THREAD

**Issue by BWPanda** [backdrop-ops/backdropcms.org#768]
As [discussed in Zulip](https://backdrop.zulipchat.com/#narrow/stream/218635-Backdrop/topic/PMC.20members) recently, there's a desire to see more information about PMC members on the [Leadership page](https://backdropcms.org/leadership):

> @alanmels:
> it's easy to recognize those PMC members who use their real names everywhere. I had to search on Andy Shillingford to find out he is, in fact, docwilmot

> @laryn:
> I think it would be good to list the perspective/constituency they are bringing to the role. The leadership page lists "core developers, back-end developers, front-end developers, site-builders, people familiar with the business of website development, people focused on user-experience, and others as identified by the community" as a starting point.

I therefore recommend updating the Leadership page with more info about PMC members:
- GitHub usernames
- The community 'perspective' they represent
- Short bio listing their contributions, reason(s) for being elected, etc.

**Comment by olafgrabienski** [comment]
I agree, more information would be helpful. There are links to the backdropcms.org profiles pages of PMC members. Would it be sufficient to add the missed information on these pages? Some provide already bio and/or Github name, others not.

**Comment by oadaeh** [comment]
I think filling out the bio more would be a great step, but I also think that more PMC specific info would be best on the leadership page.

**Comment by jackaponte** [comment]
Here's an example from the other board I'm on (though currently still technically on sabbatical from that one!): https://aspirationtech.org/about/board (A Backdrop site, too!)

Something like that could be nice. Those bios are kinda long though, and I like how comprehensive our current Leadership page is. Maybe the longer bios of PMC members (and core committers?) could be on separate page linked from Leadership, e.g. https://backdropcms.org/leadership/pmc ?

**Comment by jenlampton** [comment]
Here's a very old slide that includes constituency for some PMC members: https://backdrop-ops.github.io/slides/philosophy.html#/11

**Comment by yorkshire-pudding** [comment]
> Here's an example from the other board I'm on (though currently still technically on sabbatical from that one!): https://aspirationtech.org/about/board (A Backdrop site, too!)
> 
> Something like that could be nice. Those bios are kinda long though, and I like how comprehensive our current Leadership page is. Maybe the longer bios of PMC members (and core committers?) could be on separate page linked from Leadership, e.g. https://backdropcms.org/leadership/pmc ?

Accordion perhaps? (https://backdropcms.org/project/views_accordion / https://backdropcms.org/project/paragraphs_jquery_ui_accordion)

**Comment by stpaultim** [comment]
Another idea that came up in our meeting today was to link to the Blog post that was written to announce each new member. 

Of course, then we would need to write those blog posts. 

OK, this idea won't work today, but could be a part of the process moving forward and we could put more emphasis on getting those blogs posts done. 

(Remind me how many blogs posts I owe the group?)

**Comment by NormPlum** [comment]
Posting here on recommendation in Zulip to try to revive this issue and get it actioned.

**Comment by albanycomputers** [comment]
More transparency would be good.

**Comment by NormPlum** [comment]
@yorkshire-pudding @oadaeh @olafgrabienski have all commented here and are all website editors with the ability to implements this (according to https://backdropcms.org/website-editors), so is one of you able to action this?

**Comment by stpaultim** [comment]
I would suggest that we ask each member of the PMC and Leadership team to provide a short one paragraph bio that can be shared on the website - before Backdrop LIVE. Is this what people are looking for?

We could also post more information about the PMC itself if that is what people are looking for. 

We could also have a meet the PMC session at Backdrop LIVE.

**Comment by yorkshire-pudding** [comment]
I am happy to do content edits if needed where the content is provided.

**Comment by NormPlum** [comment]
This is what we have currently:

<img width="703" height="437" alt="Image" src="https://github.com/user-attachments/assets/c0949ea1-99bc-4888-bd45-36b15a5e1218" />

If we want to add bios, and possibly other information like usernames, community 'perspectives', etc. as people above have suggested, we may need to adjust the formatting/styling of the page as well, and not just add content...

**Comment by jenlampton** [comment]
Do we want do to the same for core committers?

**Comment by irinaz** [comment]
@NormPlum , looks like we have reasonable information for PMC members, and also for most web editors.   Which profiles are incomplete now?  Changing format for web editors page can be done only by developer, not content editor. 

@jenlampton , we also need redirect from https://backdropcms.org/leadership/pmc   to https://backdropcms.org/leadership#pmc

**Comment by jenlampton** [comment]
Redirect added ✅

**Comment by stpaultim** [comment]
@NormPlum - Each of the photos in your screenshot, comes with a link to a bio. That might not be clear to everyone.

Do we need special bios on the leadership page or should we simply expand on the bios that are linked to?

I see that you are asking for more information about each members perspectives on the community. That seems reasonable. Is there anything else in particular that you would like to see.

I'm going to start this process right now, by updating my Backdrop CMS bio to include more of the information I think your asking for. If you would like to see something that I've not provided, let me know.

**Comment by NormPlum** [comment]
> Each of the photos in your screenshot, comes with a link to a bio. That might not be clear to everyone.

I was also thinking this... I don't think adding the bios to this page is necessary, but perhaps a text link to the bio would make it more obvious you can click to read about each person.

> Is there anything else in particular that you would like to see.

I think what's been mentioned has covered everything people have suggested. If we can add text links to bios, I think I'd be happy with this and we could close it as completed.

Unless anyone else has further suggestions.

**Comment by stpaultim** [comment]
I updated my bio on BackdropCMS.org maybe other PMC members can do the same.
https://backdropcms.org/account/timtriploco

Maybe something like this, where "Profile Page" is the link?

<img width="425" height="171" alt="Image" src="https://github.com/user-attachments/assets/1cca4930-3b65-466c-aa07-593c6df82c92" />

**Comment by olafgrabienski** [comment]
Oh, I just rediscovered a blog post from April 2020 introducing me and two other new PMC members and a new core committer. The post mentions, among other things, why each member was nominated.

https://backdropcms.org/news/growing-our-community-means-growing-our-leadership

**Comment by olafgrabienski** [comment]
And here is another blog post from January 2025 which announces our newest PMC member, Irina Zaks:

https://backdropcms.org/news/a-pmc-update (scroll down to see the announcement)
